proxy.conf.js

```diff
@@ -4,9 +4,10 @@ const PROXY_CONFIG = [
             "/api",
             "/assets",
             "/search",
-            "/action"
+            "/action",
+            "/auth"
         ],
-        "target": "http://localhost:3000",
+        "target": "https://dev.sunbirdsaas.com",
```

Comment:
Why keep anything hardcoded ?
Let user give that information - this way there are no changes this value would conflict if user doesn't give.